service/mcs/mcs.go

```diff
@@ -26,6 +26,7 @@ type ClusterCost struct {
 	Deployments        []*Deployment `json:"deployments,omitempty"`
 	TotalCost          *float64      `json:"totalCost,omitempty"`
 	TotalComputeCost   *float64      `json:"totalComputeCost,omitempty"`
+	TotalEbsCost       *float64      `json:"totalEbsCost,omitempty"`
```

Comment:
```suggestion
	TotalEBSCost       *float64      `json:"totalEbsCost,omitempty"`
```

[See](https://github.com/golang/go/wiki/CodeReviewComments#initialisms).